content/it/docs/reference/glossary/container.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Container
+id: container
+date: 2018-04-12
+full_link: /docs/concepts/overview/what-is-kubernetes/#why-containers
+short_description: >
+  Una leggera e portabile immagine eseguibile che contiene il software e tutte le sue dipendenze.
```

Comment:
I would say "Una immagine leggera, protabile ed eseguibile che contiene un software e tutte le sue dipendenze". Also same thing for line #14 of this same file